js/pages/categoryPage/startPlay.js

```diff
@@ -0,0 +1,189 @@
+        if (!singleHeart.classList.contains('hide') && f === 0) {
+          singleHeart.classList.add('hide');
+          f += 1;
+        }
+      });
+    }
+  } else {
+    const header = document.querySelector('.header-main');
+
+    const divParent = document.createElement('div');
+    divParent.classList.add('hearts-container');
+
+    header.append(divParent);
+    addHeart(type);
+  }
+}
+
+function setNextCardAsActive() {
+  const resourse = JSON.parse(localStorage.getItem('statistic'));
+  const currentPage = localStorage.getItem('page');
+
+  const cards = document.querySelectorAll('.card');
+  const numOfActiveCard = getCardsOrder()[0];
+
+  const activeCard = cards[numOfActiveCard];
+  let currentCardObj;
+  let arrForSearch;
+
+  if (currentPage !== 'difficult') {
+    arrForSearch = resourse[currentPage];
+  } else {
+    arrForSearch = returnAllWords();
+  }
+  arrForSearch.forEach((i) => {
+    if (i.title.toLocaleLowerCase().replace(' ', '') === activeCard.id) {
+      currentCardObj = i;
+    }
+  });
+  setCurrentCard(currentCardObj);
+
+  new Audio(currentCardObj.audio).play();
+}
+
+function blockCardClick(blockCard) {
+  document.querySelector(`#${blockCard}`).classList.add('block-card');
+}
+function checkCorrectCard(e) {
+  const blockCard = shortName(getCurrentCard().title);
+
+  if (e.target.getAttribute('id') === blockCard) {
+    new Audio('./assets/audio/win.mp3').play();
+    editStatistics(getCurrentCard().title, 'playClick', 1);
+
+    getCardsOrder().shift();
+    setCardsOrder(getCardsOrder());
+    blockCardClick(blockCard);
+
+    if (getCardsOrder().length >= 1) {
+      addHeart('GOOD');
+      setNextCardAsActive();
+    } else {
+      addHeart('GOOD');
+      finishGame();
+    }
+  } else {
+    addHeart('BAD');
+    new Audio('./assets/audio/fail.mp3').play();
+    editStatistics(getCurrentCard().title, 'errors', 1);
+    editStatistics(getCurrentCard().title, 'playClick', 1);
+  }
+}
+function createListenersForPlay() {
+  const allCards = document.querySelectorAll('.card');
+
+  allCards.forEach((card) => {
+    card.addEventListener('click', checkCorrectCard);
+  });
+}
+
+function startPlay() {
+  const cards = document.querySelectorAll('.block-card');
+  cards.forEach((card) => {
+    card.classList.remove('block-card');
+  });
+  createArray(document.querySelectorAll('.card').length);
+  setNextCardAsActive();
+  createListenersForPlay();
+}
+
+function clickPlayBtn() {
+  const btnPlay = document.querySelector('.btn');
+  const block = document.querySelector('.play-block');
+
+  btnPlay.addEventListener('click', () => {
+    if (!block.classList.contains('hide') && localStorage.getItem('page') !== 'sections' && btnPlay.textContent === 'PLAY') {
+      hideBlockOnPlay();
+      cleanForNewGame();
+      changeTextOnBtn('REPEAT');
+
+      startPlay();
+    } else if (block.classList.contains('hide') && localStorage.getItem('page') !== 'sections' && btnPlay.textContent === 'REPEAT') {
```

Comment:
if .. else if... else.. constructions usually looks complicated and not human readable, can it be simplified?